checker/tests/ainfer-testchecker/non-annotated/README.md

```
@@ -1,21 +1,23 @@
-The files in this directory are tested while using the -Ainfer option.
+oThe files in this directory are tested while using the `-Ainfer` option.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)